src/graphics/pipeline/shaders/streamline.wgsl

```diff
@@ -14,7 +14,7 @@ struct FragmentInput {
 fn vertex(in: VertexInput) -> FragmentInput {
     return FragmentInput(
         transform * vec4<f32>(in.position, 1.),
-        abs(in.direction)
+        in.direction
```

Comment:
The input is now a color and not a direction (I forgot to change the field name here). I removed it since it makes more sense conceptually. But in practice, both approaches work well.

Do you recommend putting the function back in or simply renaming the field to `color`?

Reply:
`signed_color` would be a fitting name if we use the shader approach.

Reply:
I prefer renaming to wathever you think is right.